specification/sections/01-flag-evaluation.md

```diff
@@ -104,25 +107,38 @@ See [hooks](./04-hooks.md) for details.
 OpenFeature.getProviderMetadata();
 ```
 
-See [provider](./02-providers.md) for details.
+It must be possible to access provider metadata using a `domain`.
+If a provider has not be registered under the requested domain, the default provider metadata is returned.
+
+```typescript
+// example provider accessor
+OpenFeature.getProviderMetadata("domain-1");
```

Comment:
We might need to check SDK implementations when this spec get merged. Java already expose this [^1] and I created a follow-up for Go SDK [^2]. Needs to check other SDKs 

[^1]: https://github.com/open-feature/java-sdk/blob/main/src/main/java/dev/openfeature/sdk/OpenFeatureAPI.java#L53-L54
[^2]: https://github.com/open-feature/go-sdk/issues/247